CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 ## 6.17.0 (Unreleased)
 
+NOTES:
+* workbench: Changed `container_image` field of `google_workbench_instance` resource to modifiable. ([#20894](https://github.com/hashicorp/terraform-provider-google/pull/20894))
```

Comment:
```suggestion
* workbench: changed `container_image` field of `google_workbench_instance` resource to modifiable. ([#20894](https://github.com/hashicorp/terraform-provider-google/pull/20894))
```
Also should this line fit in the IMPROVEMENTS?

Reply:
I kinda felt the same, since you agree I moved it.